shared/package.json

```diff
@@ -29,12 +29,15 @@
         "class-validator": "^0.13.2",
         "immer": "^9.0.15",
         "lodash-es": "^4.17.21",
+        "rbush": "^3.0.1",
+        "rbush-knn": "github:mourner/rbush-knn",
```

Comment:
I tried it right now with the older npm version (instead of the newer one on github) and it seemed to have not worked:
https://github.com/mourner/rbush-knn/pull/19, opened an issue in the repo: https://github.com/mourner/rbush-knn/issues/21